src/app/trainees/state/trainees.actions.ts

```diff
@@ -24,7 +24,11 @@ export class ClearTraineesFilter {
 
 export class SearchTrainees {
   static readonly type = "[Trainees] Search";
-  constructor(public query: string) {}
+  constructor(public searchQuery: string) {}
```

Comment:
What is the need for this arguments  to be public?

Reply:
@babul3012 the args of the actions need to be public so that the `TraineesState` class can access them